api/src/server.ts

```diff
@@ -16,18 +20,29 @@ connectDB();
 app.set("port", process.env.SERVER_PORT || 7000);
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cors());
+
+app.use(cookieSession({
+  maxAge: 24 * 60 * 60 * 1000, // Day
+  name: "synced-up",
+  keys: [process.env.COOKIE_KEY_1, process.env.COOKIE_KEY_2]
```

Comment:
Is there any convention to how these keys are named? why are they env variables and not hard coded?

Reply:
The only convention I know is that they are in capitalized snake case. One of the tutorial videos said that cookie keys shouldn't be public.

Reply:
Ah cool, yes keys, I misunderstood what these were for. I though they were to name the key:value pairs but looking at session-cookie I see they are: keys
The list of keys to use to sign & verify cookie values. Set cookies are always signed with keys[0], while the other keys are valid for verification, allowing for key rotation.
:pray: